fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Avatar.kt

```diff
@@ -203,8 +203,9 @@ fun Avatar(
                         image.value(themeMode),
                         null,
                         Modifier
-                            .align(Alignment.TopStart)
-                            .offset(presenceOffset.x, presenceOffset.y)
+                            .align(Alignment.BottomEnd)
+                            // Adding 2.dp to both side to incorporate border which is an image in Fluent Android.
+                            .offset(presenceOffset.x + 2.dp, -presenceOffset.y + 2.dp)
```

Comment:
If the presenceOffset is negative and add 2.dp then mathematiaaly it will reduce right? Is this inteneded? Have you tested it.
I am just curious and i have no knowledge about it

Reply:
Yes this is tested. Basically we need to lift up the presence indicator because figma doesn't take into account the border around it which for us is a part of the image.